client/js/bar.js

```diff
@@ -0,0 +1,44 @@
+/* constructs bar graph from scratch */
+
+const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+svg.setAttribute('viewBox', '0 0 100 100');
+
+// hard coded data for bar chart for now
+const CHART_VALUES = [1, 3, 1, 2];
+const MAX_CHART_VALUES = Math.max(...CHART_VALUES);
+
+const GRAPH_HEIGHT = 100;
+const BAR_CONTAINER_HEIGHT = GRAPH_HEIGHT / CHART_VALUES.length;
+
+// bar container contains a bar. It is composed of bar padding and fill
+const BAR_PERCENT_FILL = 0.7;
+const BAR_PERCENT_PADDING = 1 - BAR_PERCENT_FILL;
+const BAR_THICKNESS = BAR_PERCENT_FILL * BAR_CONTAINER_HEIGHT;
+const BAR_UNIT_LENGTH = 100 / MAX_CHART_VALUES;
+
+// top graph padding depends on bar padding
+const GRAPH_TOP_PADDING = (BAR_PERCENT_PADDING / 2) * BAR_CONTAINER_HEIGHT;
+const GRAPH_LEFT_PADDING = 2.5;
+
+document.getElementById('graph').appendChild(svg);
+
+/**
+ * creates SVG bar chart given chart values
+ * @param {int[]} CHART_VALUES
+ */
+function createBarChart(CHART_VALUES) {
```

Comment:
parameters should be snakeCase. Also, you should move any file level const that is derived from CHART_VALUES into this function (with snakeCase) and derive them from this parameter instead.